### DISCUSSION TITLE
Error and not able to use the Voice Changer

### DISCUSSION THREAD

**Discussion by kuswahashish** [w-okada/voice-changer#1348] [Q&A]
![image](https://github.com/user-attachments/assets/59dadec5-f7d0-42bf-950a-31e2a44bdc14)

also i am not able to use the application and neither efficient instruction or documentation

**Comment by AnnoyingFox** [comment]
I think the error message may give you a good hint of the cause.
Have you tried updating your GPU drivers?
Also, you didn't fill the issue report properly. There were fields to add your operating system, version of the software, GPU, etc... those are necessary info to help you better.

About efficient info, there is a whole tutorial (which is also part of the form given when you fill in an issue but which you manually deleted). That tutorial covers everything to run the software but is as simple as decompressing the whole file into a new separated folder and double click the file "start_http.bat". You need to wait some seconds or minutes to download the necessary files the first time and then the UI should show. That's all.

Hopefully just updating your nvidia drivers fixes your issue.
